project_template_timesheet/models/project_task.py

```diff
@@ -0,0 +1,31 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models, _
+from odoo.exceptions import ValidationError
+
+
+class ProjectTask(models.Model):
+
+    _inherit = 'project.task'
+
+    @api.model
+    def _get_values_for_invisible_template_fields(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4420733)